ext/js/app/frontend.js

```diff
@@ -256,14 +257,21 @@ export class Frontend {
      * @returns {void}
      */
     _onActionScanSelectedText() {
-        void this._scanSelectedText(false);
+        void this._scanSelectedText(false, true);
```

Comment:
Are keyword arguments a thing in JavaScript? Always hard to tell what these true false are referring to

Reply:
Keyword arguments aren't a thing but using an object can essentially do the same thing.